src/Generator.cpp

```diff
@@ -790,12 +790,12 @@ gengen
             output_types.insert(OutputFileType::registration);
             output_types.insert(OutputFileType::static_library);
         } else {
-            // if emit_flags contains "stmt_viz" but not "assembly", throw an error
-            bool has_stmt_viz = std::find(emit_flags.begin(), emit_flags.end(), "stmt_viz") != emit_flags.end();
+            // if emit_flags contains "stmt_html" but not "assembly", throw an error
+            bool has_stmt_html = std::find(emit_flags.begin(), emit_flags.end(), "stmt_html") != emit_flags.end();
             bool has_assembly = std::find(emit_flags.begin(), emit_flags.end(), "assembly") != emit_flags.end();
 
-            user_assert(!has_stmt_viz || has_assembly)
-                << "Output flag `stmt_viz` requires the `assembly` flag to also be set.";
+            user_assert(!has_stmt_html || has_assembly)
```

Comment:
Is this still needed considering #7515 has landed?

Reply:
Oh, good catch! Removed.